plugin/pxr/maya/lib/usdMaya/readJob_ImportWithProxies.cpp

```diff
@@ -178,16 +178,23 @@ bool UsdMaya_ReadJobWithSceneAssembly::_ProcessProxyPrims(
 
     // Author exclude paths on the top-level proxy using the list of collapse
     // points we found.
-    if (!collapsePointPathStrings.empty()) {
+    for (const auto& entry : collapsePointMap) {
         MStatus                  status;
         UsdMayaPrimReaderContext ctx(&mNewNodeRegistry);
 
-        // Get the geom root proxy shape node.
-        SdfPath proxyShapePath = pxrGeomRoot.GetPath().AppendChild(_tokens->MayaProxyShapeNodeName);
-        MObject proxyShapeObj = ctx.GetMayaNode(proxyShapePath, false);
+        // Get the exclude ancestor's proxy shape node.
+        SdfPath excludeAncestorPath = entry.first;
+        SdfPath proxyShapePath = excludeAncestorPath.AppendChild(
+            TfToken(TfStringPrintf("%sProxy", excludeAncestorPath.GetName().c_str())));
```

Comment:
Is there a reason "Proxy" is not in the local _tokens?

Reply:
No, I can add it if you prefer!